sdk/search/Build-SearchPackages.ps1

```diff
@@ -1,7 +1,7 @@
 $scriptDir = Split-Path $MyInvocation.MyCommand.Path -Parent
 $engDir = "$scriptDir\..\..\eng\"
 
-msbuild "$scriptDir\..\..\build.proj" /t:CreateNugetPackage /p:Scope="search"
+dotnet msbuild "$scriptDir\..\..\mgmt.proj" /t:CreateNugetPackage /p:Scope=Search
 
 dotnet restore
 dotnet build "$engDir/service.proj" /p:ServiceDirectory="search"
```

Comment:
These commands are redundant, as dotnet test or dotnet pack will also do a restore and build as a prestep. So I think all that is needed is the dotnet test and pack.

Reply:
It wasn't the case when I was trying to fix this. I can try again and see if things have improved, but they were definitely broken before I added the explicit "dotnet restore". As for the other commands, I put them there on the recommendation of @chidozieononiwu.